scripts/frontend/dev-server.js

```diff
@@ -41,6 +41,7 @@ const go = async () => {
             logLevel: 'silent',
             publicPath: '/assets/javascript/',
             ignored: [/node_modules([\\]+|\/)+(?!@guardian)/],
+            writeToDisk: true,
```

Comment:
The tells webpackDevMiddleware to output its in-memory files.

We need this in order to pull `loadable-stats.server.json` using node path in `server/document.tsx`